src/Mpdf.php

```diff
@@ -1546,11 +1546,29 @@ public function __construct(array $config = [])
 		$this->selectoption = [];
 	}
 
-	public function cleanup()
-	{
-		mb_internal_encoding($this->originalMbEnc);
-		@mb_regex_encoding($this->originalMbRegexEnc);
-	}
+
+     /**
```

Comment:
Copying the comment as is does not make sense.

Reply:
sorry, don't you.. what should I do?